runtime/dart_init.cc

```diff
@@ -243,6 +244,7 @@ Dart_Isolate IsolateCreateCallback(const char* script_uri,
   // Are we running from a Dart source file?
```

Comment:
No, not yet. It might still be helpful to start with using kernel only for the flutter app first, then add loading of service isolate from kernel.